bucket/wsl-ssh-pageant.json

```diff
@@ -0,0 +1,48 @@
+{
+    "homepage": "https://github.com/benpye/wsl-ssh-pageant/",
+    "description": "A Pageant -> TCP bridge for use with WSL, allowing for Pageant to be used as an ssh-ageant within the WSL environment.",
+    "license": "BSD-2-Clause",
+    "version": "20190513.14",
+    "architecture": {
+        "64bit": {
+            "url": [
+                "https://github.com/benpye/wsl-ssh-pageant/releases/download/20190513.14/wsl-ssh-pageant-amd64.exe",
+                "https://github.com/benpye/wsl-ssh-pageant/releases/download/20190513.14/wsl-ssh-pageant-amd64-gui.exe"
+            ],
+            "hash": [
+                "84b926fd8737cb509bba9ff5c51a01838088dadaa9513addccb5a8ace4a7f7dc",
+                "0bcc64651d420e3b614e591dfba0a741cb8c04d6ec7c90078693a2af37d1b3ef"
+            ],
+            "bin": [
+                [
+                    "wsl-ssh-pageant-amd64.exe",
+                    "wsl-ssh-pageant"
+                ],
+                [
+                    "wsl-ssh-pageant-amd64-gui.exe",
```

Comment:
If this is a GUI app, please add a `shortcuts` section with it. Sorry for the delayed review.

Reply:
No problem - this isn't a 'gui' app in the sense that it'll launch some UI, both need to be started with arguments for anything to happen. The primary difference is that the GUI app is built as a Win32 rather than a console application, instead opting for a tray icon to allow you to exit it. I can change it but unless we want to be opinionated and provide some default command line args I'm not sure it makes sense.

Reply:
No, let’s leave it as is. Thanks for the detailed explanation.